### DISCUSSION TITLE
Checkbox dynamic recipients all unchecked in the submission

### DISCUSSION THREAD

**Discussion by FrDH** [solspace/craft-freeform#1351] [Questions & Answers]
In my form, I have a dynamic recipients field (rendered as checkboxes). No matter what checkbox is checked by the user, in the stored submission, all checkboxes are unchecked. Is this a bug or is it by design?

Craft 4.7.1
Freeform 4.1.13

**Comment by kjmartens** [comment]
Sorry for the delay @FrDH,

Are you referring to viewing this in the control panel or the front end template? If the latter, can you share your template code? 🙂 

Also, is it possible you changed the field option values recently or upgraded from an older version of Freeform recently?

**Comment by FrDH** [comment]
Sorry for the delay @kjmartens, I lost track of this issue and was just now reminded about it.
I was referring to the submission as it is stored/shown in the control panel. Field options weren't changed between creating the form, submitting the form and viewing the submission.
I'll do an update and see if the issue persists.
